Casks/e/emacs@pretest.rb

```diff
@@ -1,8 +1,8 @@
 cask "emacs@pretest" do
   arch arm: "arm64-11", intel: "x86_64-10_11"
```

Comment:
Looks like the related directory suffixes are `x86_64-10_12` and `x86_64-10_14` in this version.

Reply:
Thanks, I'll get around to fixing this later today.

Reply:
```suggestion
  arch arm: "arm64-11", intel: "x86_64-10_12"
```

We can just bump it to bring it in line with `emacs` and `emacs@nightly`.